internal/sirius/edit_payment_test.go

```diff
@@ -66,3 +66,70 @@ func TestEditPayment(t *testing.T) {
 		})
 	}
 }
+
+func TestEditFeeReduction(t *testing.T) {
+	t.Parallel()
+
+	pact := newPact()
+	defer pact.Teardown()
+
+	testCases := []struct {
+		name          string
+		setup         func()
+		expectedError func(int) error
+	}{
+		{
+			name: "OK",
+			setup: func() {
+				pact.
+					AddInteraction().
+					Given("I have an lpa which has a payment and fee reduction").
```

Comment:
This should work if you change the step to `I have an lpa which has a fee reduction` and the path should be `/lpa-api/v1/payments/124` and the cypress test needs to be `124`